test/scripts/e2e_subs/box-search.sh

```diff
@@ -79,4 +79,40 @@ BOX_LIST=$(${gcmd} app box list --app-id "$APPID" --max 1)
 # shellcheck disable=SC2143
 [ "$(grep -w "$BOX_LIST" <<< "$EXPECTED")" ] # actual box is in the expected list
 
+# Create and set a box in an atommic txn group:
+
+# Create:
+BOX_NAME="str:great box"
+${gcmd} app call --from "$ACCOUNT" --app-id "$APPID" --box "$BOX_NAME" --app-arg "str:create" --app-arg "$BOX_NAME" -o box_create.txn
+
+# Set:
+BOX_VALUE="str:I'm a wonderful box"
+${gcmd} app call --from "$ACCOUNT" --app-id "$APPID" --box "$BOX_NAME" --app-arg "str:set" --app-arg "$BOX_NAME" --app-arg "str:$BOX_VALUE" -o box_set.txn
```

Comment:
Perhaps it would be good to try this without the `--box` in the second tx. Should still be fine.